app/models/artist.rb

```diff
@@ -1,4 +1,6 @@
 class Artist < ActiveRecord::Base
+  has_many :songs
```

Comment:
Interesting point. I think that makes sense if we are requiring all songs to have an artist.